bootstrapvz/common/fs/logicalvolume.py

```diff
@@ -0,0 +1,30 @@
+from bootstrapvz.base.fs.volume import Volume
+from bootstrapvz.common.tools import log_check_call
+
+
+class LogicalVolume(Volume):
+
+    def create(self, image_path):
+        self.image_path = image_path
+        self.fsm.create(image_path=image_path)
+
+    def _before_create(self, e):
+        lv_size = str(self.size.bytes.get_qty_in('MiB'))
+        vg, lv = e.image_path.split('/')[2:4]
```

Comment:
this feels hacky. How will this work if we pass a "/dev/mapper/$vg-$lv" image_path ?

Maybe there is a better way to do that. Also, there is no documentation for that, maybe at least in the commit message?